var/spack/repos/spack_repo/builtin/packages/py_metatensor_learn/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+from spack.package import *
+
+
+class PyMetatensorLearn(PythonPackage):
+    """Building blocks for the atomistic machine learning models based on PyTorch and NumPy"""
+
+    homepage = "https://docs.metatensor.org"
+    pypi = "metatensor-learn/metatensor_learn-0.3.2.tar.gz"
+
+    import_modules = ["metatensor.learn"]
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.3.2", sha256="987f63228888882a6189137ddb89f913b2fde1072c3caa83a39b9f5d50388b51")
+    extends("python@3.9:")
+    variant("torch", default=False, description="With PyTorch")
```

Comment:
This variant doesn't exist with pip, why should we add it for spack?

Reply:
Are you talking about py-metatensor-torch? That's a separate package.

Reply:
Resolving as outdated.